build.gradle

```diff
@@ -32,11 +33,16 @@ subprojects {
     apply plugin: 'com.palantir.baseline-eclipse'
     apply plugin: 'com.palantir.baseline-findbugs'
     apply plugin: 'com.palantir.baseline-idea'
+    apply plugin: 'com.palantir.docker-test-runner'
     apply plugin: 'org.inferred.processors'
 
     repositories {
         jcenter()
     }
 
     sourceCompatibility = 1.7
+
+    if (System.env.CIRCLE_TEST_REPORTS) {
+        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
```

Comment:
I'll double-check. In general, it should be possible to add a Circle/Gradle task that just copies the XML artifacts to the expected Circle location, in which case I could extract the special Circle logic from the Gradle task. I'm sure this would work in the success cases, but less sure about whether or not the task would be executed if a test fails. Will play around on Circle to test.